web/src/views/DandiMetaViewer.vue

```diff
@@ -29,7 +29,27 @@
                   This dataset has not been published!
                 </v-chip>
               </v-card-title>
-              <v-card-actions>
+              <v-list dense v-if="meta.identifier" class="py-0">
+                <v-list-item>
+                  <v-list-item-content>
+                    Identifier: {{ meta.identifier }}
+                  </v-list-item-content>
+                  <!-- <v-list-item-action class="ma-0 mr-1">
+                    <v-btn icon>
+                      <v-icon>mdi-content-copy</v-icon>
+                    </v-btn>
+                  </v-list-item-action>
+                  <v-spacer /> -->
+                </v-list-item>
+                <v-list-item>
+                  <v-list-item-content>
+                    <a :href="`https://dandiarchive.org/dandiset/${meta.identifier}/draft`">
+                      {{ `https://dandiarchive.org/dandiset/${meta.identifier}/draft` }}
```

Comment:
I'm not sure how many places this base URL `https://dandiarchive.org` is hard-coded, but I'd be more comfortable if we used a variable for it.